drivers/sensor/icm42688/icm42688.h

```diff
@@ -343,8 +343,8 @@ static inline void icm42688_gyro_rads(struct icm42688_cfg *cfg, int32_t in, int3
 	*out_rads = in10_rads / (sensitivity * 180LL * 1000000LL);
 
 	/* microrad/s */
-	*out_urads = ((llabs(in10_rads) - (llabs((*out_rads)) * sensitivity * 180LL * 1000000LL))) /
-		     (sensitivity * 180LL);
+	*out_urads =
+		(in10_rads - (*out_rads * sensitivity * 180LL * 1000000LL)) / (sensitivity * 180LL);
```

Comment:
Is this change related to `test: verify accel/gyro data for icm42688`?

Reply:
Yes, it was incorrect to take the diff of the abs